zScanner/SupportFiles/Extensions/UIViewController.swift

```diff
@@ -12,4 +12,15 @@ extension UIViewController {
     var safeArea: UILayoutGuide {
         return view.safeAreaLayoutGuide
     }
+    
+    // Whenever user gonna tap somewhere else than the current view so it will dismiss the keyboard
+    func setupKeyboardHandling() {
+        let tap: UITapGestureRecognizer = UITapGestureRecognizer(target: self, action: #selector(UIViewController.dismissKeyboard))
+        tap.cancelsTouchesInView = false
+        view.addGestureRecognizer(tap)
+    }
+    
+    @objc func dismissKeyboard() {
+        view.endEditing(true)
+    }
```

Comment:
This code is used in one view controller only, why did you put it into UIViewController extension?

Reply:
I don't even know. I'm sorry I'll put it to the specific UIVIewController.